page fc & dag graph

21paradox opened the pull request:
<!-- Thanks for submitting a pull request! Please provide enough information so that others can review your pull request. -->
<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
<!-- Try to link to an open issue, bounty or submission for more information. -->


<!-- In addition to that please answer these questions: -->

**What kind of change does this PR introduce?**
page fc & dag graph
<!-- E.g. a bugfix, feature, refactoring, build related change, etc… -->

**Did you add tests for your changes?**

<!-- Note that we won't merge your changes if you don't add tests -->

**Does this PR introduce a breaking change?**

<!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->

**What needs to be documented once your changes are merged?**

<!-- List all the information that needs to be added to the documentation after merge -->
<!-- When your changes are merged you will be asked to contribute this to the documentation -->

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/conflux-chain/conflux-scan/47)
<!-- Reviewable:end -->

yqrashawn committed:
Add: remove renavate

Commit:
fc basic page

Commit:
fc basic page

yqrashawn committed:
Add: copyArtifacts from conflux-dag

yqrashawn committed:
Add: try use docker

Commit:
fc page request api

Commit:
fc page request api

Commit:
hide ellipsisItem

Commit:
hide ellipsisItem

Commit:
Merge pull request #28 from 21paradox/fix1111

hide ellipsisItem

Commit:
fc logo icon

Commit:
fc logo icon

Commit:
fix decodedData check

Commit:
fix decodedData check

Commit:
Merge branch 'dev' into pageFc

kodiakhq[bot] committed:
Page fc (#29)

<!-- Thanks for submitting a pull request! Please provide enough information so that others can review your pull request. -->
<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
<!-- Try to link to an open issue, bounty or submission for more information. -->
fc page

<!-- In addition to that please answer these questions: -->

**What kind of change does this PR introduce?**

<!-- E.g. a bugfix, feature, refactoring, build related change, etc… -->

**Did you add tests for your changes?**

<!-- Note that we won't merge your changes if you don't add tests -->

**Does this PR introduce a breaking change?**

<!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->

**What needs to be documented once your changes are merged?**

<!-- List all the information that needs to be added to the documentation after merge -->
<!-- When your changes are merged you will be asked to contribute this to the documentation -->

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/conflux-chain/conflux-scan/29)
<!-- Reviewable:end -->

yqrashawn committed:
Add: use current-slim instead of latest alpine

yqrashawn committed:
Fix: mkdir before mount

Commit:
add cfx grip in detail

Commit:
add cfx grip in detail

Commit:
Merge pull request #30 from 21paradox/pageFc

add cfx grip in detail

yqrashawn committed:
Add: WIP, dag into scan

Commit:
add entry & format

Commit:
add entry & format

Commit:
Merge pull request #31 from 21paradox/pageFc

add fc entry & format

Commit:
update i18n

Commit:
update i18n

Commit:
Merge pull request #32 from 21paradox/pageFc

update i18n & icon

yqrashawn committed:
Add: dag mouse support

yqrashawn committed:
Merge branch 'dev' into jenkins-pipeline

yqrashawn committed:
Merge branch 'dev' into removeRenovate

kodiakhq[bot] committed:
add conflux-dag.js, use docker for building (#33)

----

#

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/conflux-chain/conflux-scan/33)
<!-- Reviewable:end -->

yqrashawn committed:
Merge branch 'dev' into removeRenovate

kodiakhq[bot] committed:
Add: remove renavate (#26)

----

#

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/conflux-chain/conflux-scan/26)
<!-- Reviewable:end -->

Commit:
use notice component replace toast error

Commit:
Merge pull request #34 from 21paradox/pageFc

use notice component replace toast error

Commit:
fix fetch fetchTxDetail after txnhash update

Commit:
Merge pull request #35 from 21paradox/pageFc

fix fetch fetchTxDetail after txnhash update

yqrashawn committed:
Add: dag into blocks and blockntx

yqrashawn committed:
Add: add interactions to dag

yqrashawn committed:
Fix: dag style

yqrashawn committed:
Add: debug proxy for dag in scan

kodiakhq[bot] committed:
Fix style, anti aliasing (#40)

----

#

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/conflux-chain/conflux-scan/40)
<!-- Reviewable:end -->

yqrashawn committed:
Fix: dag style

yqrashawn committed:
Fix: block title style

kodiakhq[bot] committed:
Fix: block title style (#42)

----

#

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/conflux-chain/conflux-scan/42)
<!-- Reviewable:end -->

Commit:
add social icon & optimize search

Commit:
add status 1/2 message

Commit:
update status msg

kodiakhq[bot] committed:
fc status & input search change (#43)

<!-- Thanks for submitting a pull request! Please provide enough information so that others can review your pull request. -->
<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
<!-- Try to link to an open issue, bounty or submission for more information. -->


<!-- In addition to that please answer these questions: -->

**What kind of change does this PR introduce?**
fc status & input search change
<!-- E.g. a bugfix, feature, refactoring, build related change, etc… -->

**Did you add tests for your changes?**

<!-- Note that we won't merge your changes if you don't add tests -->

**Does this PR introduce a breaking change?**

<!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->

**What needs to be documented once your changes are merged?**

<!-- List all the information that needs to be added to the documentation after merge -->
<!-- When your changes are merged you will be asked to contribute this to the documentation -->

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/conflux-chain/conflux-scan/43)
<!-- Reviewable:end -->

Commit:
fix link pushstate

kodiakhq[bot] committed:
fix link pushstate (#44)

<!-- Thanks for submitting a pull request! Please provide enough information so that others can review your pull request. -->
<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
<!-- Try to link to an open issue, bounty or submission for more information. -->


<!-- In addition to that please answer these questions: -->

**What kind of change does this PR introduce?**
fix link pushstate
<!-- E.g. a bugfix, feature, refactoring, build related change, etc… -->

**Did you add tests for your changes?**

<!-- Note that we won't merge your changes if you don't add tests -->

**Does this PR introduce a breaking change?**

<!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->

**What needs to be documented once your changes are merged?**

<!-- List all the information that needs to be added to the documentation after merge -->
<!-- When your changes are merged you will be asked to contribute this to the documentation -->

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/conflux-chain/conflux-scan/44)
<!-- Reviewable:end -->

Commit:
stylefix

Commit:
Merge pull request #45 from 21paradox/pageFc

fix transation detail style

Commit:
update stauts msg

Commit:
Merge pull request #46 from 21paradox/pageFc

update stauts msg

Commit:
Merge branch 'master' into dev

AngeliaQ reviewed (dismissed):
> Reviewed 50 of 50 files at r1.
> *[Reviewable](https://reviewable.io/reviews/conflux-chain/conflux-scan/47)* status: :shipit: complete! all files reviewed, all discussions resolved



<!-- Sent from Reviewable.io -->

yqrashawn committed:
Fix: docker deploy

yqrashawn committed:
Merge branch 'dev' into fixScanServiceDocker

kodiakhq[bot] committed:
Fix: service docker deploy (#48)

----

#

<!-- Thanks for submitting a pull request! Please provide enough information so that others can review your pull request. -->
<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
<!-- Try to link to an open issue, bounty or submission for more information. -->


<!-- In addition to that please answer these questions: -->

**What kind of change does this PR introduce?**

<!-- E.g. a bugfix, feature, refactoring, build related change, etc… -->
Remove pm2, fix service docker image. (CMD -> ENTRYPOINT)

**Did you add tests for your changes?**

<!-- Note that we won't merge your changes if you don't add tests -->
No

**Does this PR introduce a breaking change?**

<!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->
No

**What needs to be documented once your changes are merged?**

<!-- List all the information that needs to be added to the documentation after merge -->
<!-- When your changes are merged you will be asked to contribute this to the documentation -->

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/conflux-chain/conflux-scan/48)
<!-- Reviewable:end -->

AngeliaQ reviewed (dismissed):
> Reviewed 5 of 5 files at r2.
> *[Reviewable](https://reviewable.io/reviews/conflux-chain/conflux-scan/47)* status: :shipit: complete! all files reviewed, all discussions resolved



<!-- Sent from Reviewable.io -->

yqrashawn committed:
Fix: backend server

yqrashawn committed:
Add: backto pm2

kodiakhq[bot] committed:
Add: backto pm2 (#49)

----

#

<!-- Thanks for submitting a pull request! Please provide enough information so that others can review your pull request. -->
<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
<!-- Try to link to an open issue, bounty or submission for more information. -->


<!-- In addition to that please answer these questions: -->

**What kind of change does this PR introduce?**

Use pm2 instead of docker.
<!-- E.g. a bugfix, feature, refactoring, build related change, etc… -->

**Did you add tests for your changes?**
No
<!-- Note that we won't merge your changes if you don't add tests -->

**Does this PR introduce a breaking change?**

No
<!-- If this PR introduces a breaking change, please describe the impact and a migration path for existing applications. -->

**What needs to be documented once your changes are merged?**

<!-- List all the information that needs to be added to the documentation after merge -->
<!-- When your changes are merged you will be asked to contribute this to the documentation -->

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/conflux-chain/conflux-scan/49)
<!-- Reviewable:end -->

yqrashawn reviewed (approved):
> Reviewed 47 of 50 files at r1, 1 of 5 files at r2, 4 of 4 files at r3.
> *[Reviewable](https://reviewable.io/reviews/conflux-chain/conflux-scan/47)* status: :shipit: complete! all files reviewed, all discussions resolved



<!-- Sent from Reviewable.io -->
